.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+branch = true
+include = suitcase/mongo_layout1/*
```

Comment:
These lines (3-4) should include mongo embedded and normalized and exclude their respective `_version.py` files.

Reply:
updated